### DISCUSSION TITLE
Any good source code examples to feel the resilence4j

### DISCUSSION THREAD

**Discussion by nagkumar** [resilience4j/resilience4j#2006] [General]
Any github examples that demo the power of resilience and how to get there using resilience4j?

I do see documentation https://resilience4j.readme.io/docs/examples that gives examples... but can't find any github source code that can be used as is..

**Comment by RobWin** [comment]
https://github.com/resilience4j/resilience4j-spring-boot3-demo

**Comment by nagkumar** [comment]
Thank You.

Follow-up question, can resilence4j be used in any Java application or works only with spring applications?

**Reply by RobWin** [reply]
Can be used in any Java application.
But then you need to configure everything by yourself and implement support for Annotations by yourself.
In Spring Boot you get a lot of free work as part of the auto configurations.
